starknet_py/serialization/factory_test.py

```diff
@@ -95,6 +100,12 @@ def test_getting_payload_serializer():
     )
 
 
+def test_getting_payload_serializer_v2():
```

Comment:
Could we also add explicit tests for `serializer_for_event` for V1 and V2? An approach similar to [this](https://github.com/software-mansion/starknet.py/blob/development/starknet_py/serialization/serialization_test.py#L124), could be done for actual contracts.